spec/support/auth0_helpers.rb

```diff
@@ -0,0 +1,18 @@
+module Auth0Helpers
```

Comment:
These methods were all lifted from CCS RMI so I am commenting them in as I need them but they do not end up as commented out.